juicy-html.html

```diff
@@ -218,8 +218,15 @@
             if (model === null || !arrayOfElements) {
                 return;
             }
-            for (var childNo = 0; childNo < arrayOfElements.length; childNo++) {
-                arrayOfElements[childNo].model = model;
+            for (let childNo = 0; childNo < arrayOfElements.length; childNo++) {
+                const elem = arrayOfElements[childNo];
+                elem.model = model;
+
+                //mirrored implementation of this fix: https://github.com/Polymer/polymer/pull/4537/files
+                const firstElemChild = elem.firstElementChild;
+                if (firstElemChild && firstElemChild.getAttribute("is") === elem.localName) {
```

Comment:
Are we fine to do this for

Reply:
How do you know Polymer implements it in `dom-*` elements only? Looking [here](https://github.com/Polymer/polymer/pull/4537/files) it seems that they do it for any element names.

Reply:
Look here https://github.com/Polymer/polymer/pull/4537/files#diff-b0cd31494957f8cabb844cccbe72fe2fL62

Reply:
This is why it works only for `dom-*` elements

Reply:
Can you paste a link to a line?

Reply:
Lets hope `<foo><bar is="foo">` will not become a feature by that time.